starters/cloudbuild.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - name: maven
+    entrypoint: mvn
+    args: ['-f', './pamelalozano', 'package']
+  - name: maven
+    entrypoint: mvn
+    args: ['-f', './aroquev/portfolio', 'package']
+  - name: maven
+    entrypoint: mvn
+    args: ['-f', './ayup/portfolio', 'package']
+
```

Comment:
I don't think this is working with cloudbuild unfortunately :/

Reply:
Should we just stop running cloudbuild for starter projects?